tests/test_apod.py

```diff
@@ -0,0 +1,53 @@
+from http import HTTPStatus
```

Comment:
A unit test should be independent, isolated, and repeatable. Read more about the unit testing idea, please (something like [this](https://medium.com/chris-nielsen/so-whats-a-good-unit-test-look-like-71f750333ac0#:~:text=Good%20unit%20tests%20are%20independent%20and%20isolated&text=They%20certainly%20don't%20rely,file%20system%20or%20a%20database.)) and try to fix yours.

Reply:
Hi, fixed tests. 
https://github.com/mstoro/nasa_api/pull/1/commits/6981799217867c0f50baa0dd1d387a94bf8aa070